src/components/SceneStats/SceneStats.container.ts

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux'
+import { RootState } from 'modules/common/types'
+import { fetchSceneStatsRequest, FetchSceneStatsRequestAction, FETCH_SCENE_STATS_REQUEST } from 'modules/stats/action'
+import { getStatsByDeploymentId, getLoading } from 'modules/stats/selectors'
+import { MapStateProps, MapDispatchProps, MapDispatch, OwnProps } from './SceneStats.types'
+import SceneStats from './SceneStats'
+
+const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => ({
+  stats: getStatsByDeploymentId(state)[ownProps.deployment.id],
+  isLoading: getLoading(state)
+    .filter(action => action.type === FETCH_SCENE_STATS_REQUEST)
+    .some(action => (action as FetchSceneStatsRequestAction).payload.base === ownProps.deployment.base)
```

Comment:
Maybe this can be a selector on its own? if not, you can just use some for the entire condition instead of looping twice right?

Reply:
done 👍